File: testprojects/csharp/benchmark/domain/Indirect/AlowedFrom/AccessStaticVariableIndirect_VarVar.cs
namespace Domain.Indirect.AllowedFrom
{
	public class AccessStaticVariableIndirect_VarVar
	{
        private Domain.Indirect.Intermediate.BackgroundService bckg;

        private string test;

        public void Case5()
        {
            bckg = new Domain.Indirect.Intermediate.BackgroundService();
            test = bckg.serviceOne.sName;
        }
	}
}

File: testresources/java/accuracy/src/domain/indirect/allowedfrom/AccessStaticVariableIndirect_VarVar.java
package domain.indirect.allowedfrom;

import domain.indirect.intermediate.BackgroundService;

public class AccessStaticVariableIndirect_VarVar {
	private BackgroundService bckg;
	private String test;

	public AccessStaticVariableIndirect_VarVar(){
		bckg = new BackgroundService();
	}
		//  Indirect access of a static attribute
		@SuppressWarnings("static-access")
		public String case5() {
			test = bckg.serviceOne.sName;
			return test;
		}
}
